src/casync-tool.c

```diff
@@ -262,12 +263,16 @@ static int parse_what_selector(const char *arg, enum arg_what *what) {
                 *what = WHAT_BLOB_INDEX;
         else if (streq(arg, "directory"))
                 *what = WHAT_DIRECTORY;
-        else {
+        else if (streq(arg, "help")) {
+                printf("Allowed --what= selectors:\n"
+                       "archive-index\nblob\nblob-index\ndirectory\n");
```

Comment:
not that it matters but on each newline i'd actually add a real newline too...

Reply:
Done.